docs/learn.rst

```diff
@@ -143,19 +172,19 @@ Independent of the required action (writing an answer yourself or choosing one o
 
 #. **Read the question carefully**.
 #. Write the answer or choose one of the provided options.
-#. When you are ready to submit, click the **Check answer** button.
+#. When you are ready to submit, click the **Check** button.
 
-   * If the answer is correct and a check mark appears, click the **Next question** button to proceed.
-   * If the answer is incorrect, click the **Get a hint** button, read the suggestions, and try to answer again.
-   * The progress bar below the exercise is taking into account only the most recent answers you give, and that means that you need to have the required number of correct answers (*check marks*) **in the row** for the exercise to be considered completed.
+   * When the answer is correct and a |green-check| icon appears in the progress bar below the exercise, click the **Next** button to proceed.
+   * If the answer is incorrect, click the **Use a hint** button, read the suggestions, and try to answer again.
```

Comment:
note that they don't _have_ to use a hint at this point - they can just try again